internal/commands/build_test.go

```diff
@@ -77,6 +76,7 @@ func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 
 				command.SetArgs([]string{"--builder", "my-builder", "image"})
 				h.AssertNil(t, command.Execute())
+				h.AssertContains(t, outBuf.String(), "Warning: You are building an image named 'image'. If you mean it as an app directory path, run 'pack build <args> --path image'")
```

Comment:
Note:
I've added this assertion only to this case since asserting it in all applicable cases would be too noisy. Please let me know if you have any thoughts.